.evergreen/config_generator/components/kms_divergence_check.py

```diff
@@ -0,0 +1,50 @@
+from shrub.v3.evg_command import EvgCommandType
+from shrub.v3.evg_command import s3_put
+from shrub.v3.evg_task import EvgTask
+
+from config_generator.etc.function import Function
+from config_generator.etc.function import merge_defns
+from config_generator.etc.utils import bash_exec
+
+
+class KmsDivergenceCheck(Function):
+    name = "kms-divergence-check"
+    commands = [
+        bash_exec(
+            command_type=EvgCommandType.TEST,
+            working_dir="mongoc",
+            include_expansions_in_env=["distro_id"],
+            script="""\
+                set -o errexit
```

Comment:
That makes sense. I need to finally shift my default to "use separate scripts rather than putting everything inline in the Evergreen project files". I'll make these updates as you suggested.